include/object_constants.h

```diff
@@ -197,6 +197,19 @@
     #define BOBOMB_BUDDY_HAS_NOT_TALKED 0
     #define BOBOMB_BUDDY_HAS_TALKED 2
 
+/* Fish */
+    /* oAction */
+    #define FISH_INIT 0
+    #define FISH_ACTIVE 1
+    #define FISH_RESPAWN 2
+
+/* Blue_Fish */
+    /* oAction */
+    #define BLUE_FISH_SPAWN 0
+    #define BLUE_FISH_RANDOM_YAW 1
+    #define BLUE_FISH_RANDOM_PITCH 2
+    #define BLUE_FISH_RANDOM_YAW_2 3
```

Comment:
I don't like having RANDOM_YAW and RANDOM_YAW_2, it's confusing and not descriptive. What's the difference between these?

Reply:
Also you forgot _ACT_, i.e. `BLUE_FISH_ACT_RANDOM_YAW`

Reply:
Random yaw and random yaw 2 are near identical

Reply:
maybe (BLUE_FISH_ACT_)`DIVE`, `TURN`, `ASCEND`, and `TURN_BACK` are more descriptive.
unk100 is always +-0x800, so the fish turns 0x8000 (180 degrees) during 1, and again during 3. rng just decides left or right